docs/USER_GUIDE.md

```diff
@@ -279,9 +286,35 @@ To add more users to the application:
 4. Use this page to create or manage users. For more information, consult the
    [Managing Users in User Pools Guide].
 
-In order to use the API, you will need to authenticate the requests using the
-User Pool. For more information, consult the [Cognito REST API integration
-guide].
+### Making authenticated API requests
+
+If you need to use the API directly, you will need to authenticate the requests
+using the User Pool. After resetting the password via the UI, you can make
+authenticated requests using the AWS CLI:
+
+1. Note the _CognitoUserPoolId_ and _CognitoUserPoolClientId_ parameters
+   displayed in the _Outputs_ tab for the stack, and take note of cognito user
+   email and password.
+2. Generate a token by running this command:
+
+   ```sh
+   aws cognito-idp admin-initiate-auth \
+     --user-pool-id <user-pool-id> \
+     --client-id <user-pool-client-id> \
+     --auth-flow ADMIN_NO_SRP_AUTH \
+     --auth-parameters '{"USERNAME":"<user-email>","PASSWORD":"<your-password"}'
+   ```
+
+3. Take note of the `IdToken` from the previous command. Now you can run an
```

Comment:
3. Use the `IdToken` generated by the previous command to make an authenticated
    request to the API. The API base URL can be obtained from the _ApiUrl_ in the
    _Outputs_ tab for the CloudFormation stack. For example, the following command
    will show the matches in the deletion queue:
 
   ```sh
   curl <api-url>/v1/queue -H "Authorization: Bearer <id-token>"
   ```

Reply:
couldn't make it a suggestion since it's multi-line :(

Reply:
Done